Lib/fontTools/varLib/__init__.py

```diff
@@ -31,9 +31,22 @@
 from fontTools.varLib import designspace, models, builder
 from fontTools.varLib.merger import merge_tables, Merger
 import warnings
+import re
 import os.path
 from argparse import ArgumentParser
 
+CLASS_VALS = {
+	'Thin': 100,
```

Comment:
Ah! All that axes/map stuff is relatively new in mutatormath, thanks for bringing it to my attention. I think we can definitely use it here. As for the custom axis, that's this old issue: https://github.com/googlei18n/fontmake/issues/144.